schemas/resources/schemas/grid/v0.0.1/schema.json

```diff
@@ -733,22 +737,22 @@
     "DimensionScaleXUnitPatternDMS": { 
       "$comment": "Checks the DimensionScale's Unit element values pattern for Degrees-Minutes-Seconds. The value's format ##-##-##.### and Milliseconds are optional.",
       "if": {"properties": {"X": {"properties": {"Unit": {"enum": ["Degrees-Minutes-Seconds"]}}}}},
-      "then": {"properties": {"X": {"properties": {"Minimum": {"pattern": "^[0-9]{2}-[0-9]{2}-[0-9]{2}\\.?\\d*?$"},
-        "Maximum": {"pattern": "^[0-9]{2}-[0-9]{2}-[0-9]{2}\\.?\\d*?$"}}}}}
+      "then": {"properties": {"X": {"properties": {"Minimum": {"pattern": "^-?(360-00-00\\.?[0]*)$|-?(3[0-5][0-9]-[0-9]{2}-[0-9]{2}\\.?\\d*)$|[0-2][0-9][0-9]-[0-9]{2}-[0-9]{2}\\.?\\d*?$"},
```

Comment:
Does it matter if the minimum here is -360 when the minimum listed above is -359?

Reply:
No, metadata.json is just an example file.  -360 is the correct value.